catalogs/hadoop-common/src/main/java/org/apache/gravitino/catalog/hadoop/fs/Constants.java

```diff
@@ -0,0 +1,29 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ *  or more contributor license agreements.  See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.  The ASF licenses this file
+ *  to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.apache.gravitino.catalog.hadoop.fs;
```

Comment:
Can you rename the package to `o.a.g.catalog.hadoop.common` for these files?

Reply:
I plan to do so before, these changes will bring too many changes to other modules. For instance, changing this package will make all `FilesystemProvider` implementations scattered in the bundle modules. Another point is that the SPI file is `org.apache.gravitino.catalog.hadoop.fs.FileSystemProvider`, I'm hesitant to make these changes. Would this be acceptable to you? If so, I will make the necessary changes.

Reply:
OK, we can do this in another PR.